src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index_ja.properties

```diff
@@ -88,6 +88,8 @@ Gerrit\ Connection\ Setting=\
     Gerrit\u63a5\u7d9a\u8a2d\u5b9a
 Notification\ Level=\
     \u901a\u77e5\u30ec\u30d9\u30eb
+Silent\ Level=\
```

Comment:
No need to add default translations, if a key is missing for the current lang, the default lang will be used. Better to not have them here, so a translator can find what is missing.